client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -131,14 +131,14 @@
 import java.util.StringJoiner;
 import java.util.stream.Collectors;
 
-final class RequestConverters {
-    static final XContentType REQUEST_BODY_CONTENT_TYPE = XContentType.JSON;
+public final class RequestConverters {
+    public static final XContentType REQUEST_BODY_CONTENT_TYPE = XContentType.JSON;
 
     private RequestConverters() {
         // Contains only status utility methods
     }
 
-    static Request cancelTasks(CancelTasksRequest cancelTasksRequest) {
+    public static Request cancelTasks(CancelTasksRequest cancelTasksRequest) {
```

Comment:
I talked to @hub-cap about these - these aren't going to stay public. We'll un-public them in a few PRs.

Reply:
yes the thought is to move all of the client specific request converters into their own classes and then put them in the same package, removing the `public`. The only public ones will be the methods shared by more than one of the request converters.